__tests__/product/product.variables.js

```diff
@@ -419,10 +419,40 @@ const newModel = {
   },
 }
 
+const newMaterial = {
+  name: [
+    { lang: "uk", value: "Тестовий матеріальчик" },
```

Comment:
Please, here single quotes instead of double qoutes

Reply:
done